lazors/lazors.py

```diff
@@ -1,4 +1,8 @@
 #!/usr/local/bin/python3
+from gevent import monkey as MONKEE
+MONKEE.patch_all()
```

Comment:
HOOOOOOOOOOOO

Reply:
`#!/usr/bin/env python

from bananas import utils

if utils.has_banana():
    print "You have a banana!`